helm/designate-certmanager-webhook/values.yaml

```diff
@@ -22,6 +22,7 @@ openstack:
   username: ""
   password: ""
   project_id: ""
+  project_name: "" # Use project_id OR project_name
```

Comment:
If only one of those variables should be set at a time I would change it to one having priority over the other.
E.g. Only setting `OS_PROJECT_ID` if both values are given :)

Reply:
IMHO this is overly complicated to implement and error prone (e.g. what happens if only `OS_PROJECT_ID` is given. I example, nothing would be insert into the Secret).

Reply:
When you only want one of both values this can very easily be solved with an `if-elif` statement.
Then if only one of both is given it would just take that one

Reply:
ok, can you check if that's what you want? :)

Reply:
Yes that is basically what I want.
You can make at a bit easier to read though if you just do like a
```
if id 
    ID
else if name
    NAME
else
    fail
end
```
This way we don't need to have this `and` (which always looks super complicated to me with helm ^^') and we can get rid of 1 if

Reply:
Damn, yes this is indeed simpler. Thanks! :)